main.go

```diff
@@ -49,10 +49,6 @@ func main() {
 		level.Error(logger).Log("err", "-token is required")
 		os.Exit(1)
 	}
-	if len(serviceIDs) <= 0 {
-		level.Error(logger).Log("err", "at least one -service ID is required")
-		os.Exit(1)
-	}
```

Comment:
This is fine, but please denote that -service is optional in the help text, e.g.

```go
fs.Var(&serviceIDs, "service", "Specific Fastly service ID only (optional, repeatable)")
```